src/java/picard/analysis/CollectRawWgsMetrics.java

```diff
@@ -68,8 +68,8 @@
             "</pre>" +
             "<hr />" +
             "Please see " +
-            "<a href='https://broadinstitute.github.io/picard/picard-metric-definitions.html#CollectWgsMetrics.WgsMetrics'>" +
-            "the WgsMetrics documentation</a> for detailed explanations of the output metrics." +
+            "<a href='https://broadinstitute.github.io/picard/picard-metric-definitions.html#CollectWgsMetrics.WgsMetricsWithNonZeroCoverage'>" +
```

Comment:
so confused...this is CollectRawWgsMetrics.java, why are you sending the reader to WgsMetricsWithNonZeroCoverage?